app/libs/Utils/HTMLHelpers.php

```diff
@@ -0,0 +1,145 @@
+            $attributesString .= " {$key}=\"{$value}\"";
+        }
+        return "<link rel=\"stylesheet\" href=\"{$url}\"{$attributesString}>";
+    }
+}
+
+if (!function_exists('script_to')) {
+    function script_to($path, $attributes = [])
+    {
+        $url = asset($path);
+        $attributesString = '';
+
+        foreach ($attributes as $key => $value) {
+            $attributesString .= " {$key}=\"{$value}\"";
+        }
+
+        return "<script src=\"{$url}\"{$attributesString}></script>";
+    }
+}
+
+if (!function_exists('button_to')) {
+    function button_to($text, $attributes = [])
+    {
+        $attributesString = '';
+
+        foreach ($attributes as $key => $value) {
+            $attributesString .= " {$key}=\"{$value}\"";
+        }
+
+        return "<button{$attributesString}>{$text}</button>";
+    }
+}
+
+if (!function_exists('link_to')) {
+    function link_to($url, $title = null, $attributes = [], $secure = null)
+    {
+        $url = empty($url) ? '#' : url($url, [], $secure);
+        $attributesString = '';
+
+        foreach ($attributes as $key => $value) {
+            $attributesString .= " {$key}=\"{$value}\"";
+        }
+        return "<a href=\"{$url}\"{$attributesString}>{$title}</a>";
+    }
+}
+
+if (!function_exists('radio_to')) {
+    function radio_to($name, $value, $checked = false, $attributes = [])
+    {
+        $attributesString = '';
+
+        foreach ($attributes as $key => $valueAttribute) {
+            $attributesString .= " {$key}=\"{$valueAttribute}\"";
+        }
+
+        $isChecked = $checked ? 'checked' : '';
+        $html = "<input type=\"radio\" name=\"{$name}\" value=\"{$value}\" {$isChecked}{$attributesString}>";
+
+        return $html;
+    }
+}
+
+if (!function_exists('select_to')) {
+    function select_to($name, $options = [], $selected = null, $attributes = [])
+    {
+        $attributesString = '';
+
+        foreach ($attributes as $key => $value) {
+            $attributesString .= " {$key}=\"{$value}\"";
+        }
+
+        $html = "<select name=\"{$name}\"{$attributesString}>";
+
+        foreach ($options as $value => $label) {
+            $isSelected = $value == $selected ? 'selected' : '';
+            $html .= "<option value=\"{$value}\" {$isSelected}>{$label}</option>";
+        }
+
+        $html .= "</select>";
+
+        return $html;
+    }
+}
+
+if (!function_exists('form_open')) {
+    function form_open($url, $method = 'POST', $attributes = [])
+    {
+        $attributesString = '';
+
+        foreach ($attributes as $key => $value) {
+            $attributesString .= " {$key}=\"{$value}\"";
+        }
+
+        $csrfField = csrf_field();
+        $methodField = '';
+
+        if (strtoupper($method) !== 'POST') {
+            $methodField = method_field($method);
+            $method = 'POST';
+        }
+
+        return "<form action=\"" . url($url) . "\" method=\"{$method}\"{$attributesString}>{$csrfField}{$methodField}";
```

Comment:
to append token u need to check if method is not GET
see https://github.com/LaravelCollective/html/blob/cb9092527e8b6cd8f9a69aedff70a3a3d95228e5/src/FormBuilder.php#L1253

Reply:
done